conbench/app/_plots.py

```diff
@@ -35,22 +38,19 @@ def get_outliers(self, benchmarks):
         return outliers, outlier_ids, outlier_names
 
     def get_history_plot(self, benchmark, run, i=0):
-        history = self._get_history(benchmark)
-        if history:
```

Comment:
`if history` had unclear purpose especially not knowing what `history` even was (it was too hard to infer the intent). I changed this to `if len(samples) > 0` below. It's about details like this to make code easier to read.